internal/dataplane/translator/translator.go

```diff
@@ -74,11 +74,12 @@ type LicenseGetter interface {
 // equivalent Kong objects and configurations, producing a complete
 // state configuration for the Kong Admin API.
 type Translator struct {
-	logger        logr.Logger
-	storer        store.Storer
-	workspace     string
-	licenseGetter LicenseGetter
-	featureFlags  FeatureFlags
+	logger           logr.Logger
+	storer           store.Storer
+	workspace        string
+	licenseGetter    LicenseGetter
+	featureFlags     FeatureFlags
+	ingressClassName string
```

Comment:
Could we add some comments on why we add an `ingressClassName` in the config translator? Are we going to create a separate translator that translate the k8s resources in the snapshot into the fallback config? If so, I think this is not so related in this single PR and could be put into the successive PRs.

Reply:
That was passed to `Translator` to make it possible to create `store.Store` (as its constructor requires `store.CacheStores`, but also ingress class name). I've changed the implementation to not pass the ingress class by extending `store.Storer` interface with `UpdateCache` method. PTAL https://github.com/Kong/kubernetes-ingress-controller/pull/5993/commits/3cabb4514cb3d42a3667d0dc62de277f7af3837e